packages/indexer-common/src/indexing-status.ts

```diff
@@ -93,10 +93,11 @@ export class IndexingStatusResolver {
             }
           }`
 
-    try {
+    const executeQuery = async () => {
```

Comment:
Maybe we can use a more precise name, something similar to `queryIndexingStatuses`?